scaleway/helpers.go

```diff
@@ -349,3 +349,11 @@ func regionSchema() *schema.Schema {
 func getRandomName(prefix string) string {
 	return namegenerator.GetRandomName("tf", prefix)
 }
+
+// diffSuppressFuncForRandomName is a generic DiffSuppressFunc to avoid changing randomly generated names.
+func diffSuppressFuncForRandomName(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
nit: `RandomNameDiffSuppressFunc` ?

Reply:
The field in TF is called `DiffSuppressFunc` so I find it logical to have it first in the name of the function. But I guess it's not that important...